src/test/kotlin/no/nav/syfo/integration/kafka/journalpost/JournalpostHendelseConsumerTest.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.syfo.integration.kafka.journalpost
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import no.nav.helse.arbeidsgiver.bakgrunnsjobb.BakgrunnsjobbRepository
+import no.nav.syfo.integration.kafka.joarkLocalProperties
+import no.nav.syfo.kafkamottak.InngaaendeJournalpostDTO
+import no.nav.syfo.repository.DuplikatRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+internal class JournalpostHendelseConsumerTest {
```

Comment:
Trenger vel ikke være internal?

Reply:
Når intellij genererer tester for meg så kommer internal med automatisk, så jeg har egentlig ikke tenkt så mye rundt det. Det kan forsåvidt være nyttig å bruke internal for å ikke lage avhengigheter mellom to tester. Er det noen grunner til å ikke bruke internal?

Reply:
Aha. Visste ikke at det kom fra autogenerering. 

`internal` stopper vel ikke avhengighet mellom tester? Det forhindrer bare bruk utenfor `syfoinntektsmelding`, hvis noen skulle importert den. Det beste er jo å ikke ha noe i koden som ikke bidrar med noe.